metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Analysis.java

```diff
@@ -105,7 +98,7 @@
     @ApiModelProperty(position = 6, dataType = "java.lang.String", example = "[url1, url2]",
             notes = "URL(s) to the reference sequence(s): either a single URL to an assembly/transcriptome, or a comma-separated list of gene sequences")
     @JsonProperty
-    @ManyToMany
+    @ManyToMany(cascade = {CascadeType.PERSIST, CascadeType.MERGE})
```

Comment:
As discussed with @selvaebi, this cascading needs to be ditched in favor of storing the reference sequence manually, in order to avoid duplicate issues

Reply:
Fixed in https://github.com/EBIvariation/metadata-ws/pull/50/commits/767b4d1a20eda626c547e83b812b3ea353078d00